test/e2e/scalability/density.go

```diff
@@ -64,6 +64,9 @@ var MaxContainerFailures = 0
 // Maximum no. of missing measurements related to pod-startup that the test tolerates.
 var MaxMissingPodStartupMeasurements = 0
 
+// Number of nodes in the cluster (computed inside BeforeEach).
+var NodeCount = 0
```

Comment:
This shouldn't be public.

s/NodeCount/nodeCount/

Reply:
Done